# tissue_classification_evaluation
The idea of this project was to evaluate genomic data normalization/transformation by performing cross-dataset validation. This needed to be done to evaluate different data transformations that remove platform effect/signal. We used two datasets containing genomic data (gene expression in this case, but the method is not limited to that particular data type), specifically GTEX project (RNA-Seq) and tissue atlas (microarray). For each of the transformations, we trained a number of tissue classifiers on one dataset and tested them on the second dataset, then visa versa. Essentially, this amounts to cross-platform validation. I used a variety of classifiers (linear, random forest, etc.) here. The classification that provided the best accuracy (trained on the first dataset and tested on the second) suggests that the data transformation used for that test is the best at removing the dataset/platform/batch effect.
scikit-learn libraries were used. 
